staging/velero/README.md

```diff
@@ -0,0 +1,107 @@
+# Velero-server
+
+This helm chart installs Velero version v1.0.0
+https://github.com/heptio/velero/tree/v1.0.0
+
```

Comment:
delete one line

Reply:
So this is from the source repository, when I make an upstream PR for this I would rather not include changes like this yeah?

Reply:
I've split the commits to better show the old vs new, let me know if this is still an issue.